test/unit/widgets/__snapshots__/peoplePicker.test.ts.snap

```diff
@@ -1,143 +0,0 @@
-// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
Did you delete this on purpose?

Reply:
Yes, because it's dependent on tests in peoplePicker.test.ts. I've commented out all the tests in peoplePicker.test.ts for now, to not interfere with the work of @SharonStrats. She'll simply add the snapshot anew when she fixes peoplePicker.test.ts after rebasing on master.